Retrospectives/Lean_Coffee.md

```diff
@@ -0,0 +1,50 @@
+# Lean Coffee
+
+```
+"Conversations are directed and productive because the agenda for the meeting was democratically generated." - leancoffee.org
+```
+
+### Smell that sweet roast in the morning.
+<img src="https://media.giphy.com/media/3oz8xyc6SlRBhKUhoI/giphy.gif" width="250" alt="Coffee cat" />
+
+## What you need
+- A board with Four (4) columns on it; whether that be Trello, Virtual Whiteboard, Piece of Paper with Post-Its.
```

Comment:
```suggestion
- A board with four (4) columns on it; whether that be Trello, a virtual whiteboard, a piece of paper with post-its.
```
Sorry, there was too many common nouns capitalised here for me not to intervene :P